src/ray/rpc/worker/core_worker_client_pool.h

```diff
@@ -82,7 +69,7 @@ class CoreWorkerClientPool {
     CoreWorkerClientEntry() {}
     CoreWorkerClientEntry(ray::WorkerID worker_id,
                           std::shared_ptr<CoreWorkerClientInterface> core_worker_client)
-        : worker_id(worker_id), core_worker_client(core_worker_client) {}
+        : worker_id(worker_id), core_worker_client(std::move(core_worker_client)) {}
```

Comment:
Consider move `worker_id`, even if it's POD now, we could have non-POD members in the future.